cmd/terrad/main.go

```diff
@@ -141,6 +141,8 @@ func persistentPreRunEFn(context *server.Context) func(*cobra.Command, []string)
 
 		rootDir := viper.GetString(flags.FlagHome)
 
+		// load application DBDir and set wasm DBDir
+		wasmconfig.DBDir = filepath.Base(context.Config.DBDir()) + "/wasm"
```

Comment:
default DBDir is data/wasm, so wasmconfig.DBDir becomes data/wasm/wasm?

Reply:
context.Config.DBDir() refers application database store `db_dir = "data"` in config.toml. so default DBDir in current context is "data"